spk/tvheadend/Makefile

```diff
@@ -14,7 +14,7 @@ DSM_UI_DIR = app
 export TVH_VERS
 
 DEPENDS = cross/busybox cross/$(SPK_NAME) cross/zap2epg
-WHEELS = requests
+WHEELS = certifi chardet requests urllib3
```

Comment:
please use a requirements.txt file for the wheels 
and pin there each package to an explicit version

Reply:
done.